core/src/test/scala/org/broadinstitute/dsde/rawls/googleProject/GoogleProjectRegistrationRepositorySpec.scala

```diff
@@ -86,4 +86,59 @@ class GoogleProjectRegistrationRepositorySpec extends AnyFlatSpec with TestDrive
 
   }
 
+  it should "retrieve a google project registration by id" in withDefaultTestDatabase {
+    val repo = new GoogleProjectRegistrationRepository(slickDataSource)
+    val googleProjectReg = makeGoogleProjectRegistration(testData.testProject1)
+
+    Await.result(repo.registerGoogleProject(googleProjectReg), Duration.Inf)
+    val result = Await.result(repo.getGoogleProjectRegistration(googleProjectReg.googleProjectId), Duration.Inf)
+
+    result match {
+      case Some(project) => assertResult(googleProjectReg)(project)
+      case None          => fail(s"Expected Some(${googleProjectReg}) but got None")
+    }
```

Comment:
totally optional; this is for your learning the codebase and frameworks. If this test class added the `org.scalatest.matchers.should.Matchers` trait (`class GoogleProjectRegistrationRepositorySpec extends AnyFlatSpec with TestDriverComponent with Matchers`), then you could write this using the [matchers DSL](https://www.scalatest.org/user_guide/using_matchers), which can be terser and arguably more readable:
```scala
result should contain(googleProjectReg)
```

I especially note this is optional feedback because you may not want to mix assertion styles within the same class.

Edited to add: I see you using Matchers in other tests! So you probably know all this already.